django/db/backends/postgresql/base.py

```diff
@@ -89,6 +107,7 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         "IPAddressField": "inet",
         "GenericIPAddressField": "inet",
         "JSONField": "jsonb",
+        "NullBooleanField": "boolean",
```

Comment:
`NullBooleanField` only exists for historical migrations - do we need to add it here?

https://github.com/django/django/blob/d3677043fce0fec5d082e8cae142d5e5df35a2b5/django/db/models/fields/__init__.py#L2196-L2220

I think since 3b9fe906bf28d2e748ce4d9a1af5fbcd5df48946 its name will no longer be used in this map?

Reply:
Yeah note sure why I kept it, thanks!